components/raftstore-v2/src/fsm/peer.rs

```diff
@@ -305,6 +305,10 @@ impl<'a, EK: KvEngine, ER: RaftEngine, T: Transport> PeerFsmDelegate<'a, EK, ER,
                         .peer_mut()
                         .on_request_split(self.store_ctx, request, ch)
                 }
+                PeerMsg::RequestHalfSplit { request, ch } => self
```

Comment:
It seems it's used by test purpose only. It would be better to make it test only.

Reply:
I guess it's not for test purpose. The request is lanuched by pt-ctl.